meta.yml

```diff
@@ -4,7 +4,7 @@ kit:
   applications:
     - name: "PetClinic on Docker"
       manifest: https://raw.github.com/qubell-bazaar/component-petclinic-docker/master/petclinic.yml
-    - name: "WAR on Docker"
+    - name: "Web application builder on Docker"
```

Comment:
disregard, I added this app from your fork, not realizing we have absolute paths in the metafile